### PR TITLE
fixed minor issues across multiple pages

### PR DESCRIPTION
**Pull request by schoudhury**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by pratmit — DISMISSED** [review]

**Review by pratmit — DISMISSED** [review]

**Review by pratmit — DISMISSED** [review]

**Review by prasadsatamharness — DISMISSED** [review]

File: src/components/HomepageFeatures/data/featureListData.tsx [hunk 1]
```diff
@@ -9,15 +9,13 @@ export const featureList: CardItem[] = [
     icon: "img/icon_code.svg",
     description: <>Accelerate development with security at scale.</>,
     link: "docs/category/get-started-with-code",
-    type: [docType.Documentation],
```
Comment (wei-harness) [inline src/components/HomepageFeatures/data/featureListData.tsx:12]: @schoudhury i see you removed all the doc icons from the `Get Started` section, i assume that was on purpose?

**Review by wei-harness — DISMISSED** [review]

**Review by rohanmaharjan100 — APPROVED** [review]

**Review by dewan-ahmed — APPROVED** [review]
LGTM for the CI part of the changes.

Needs to resolve conflicts in `.github/CODEOWNERS`

**Review by SushrutHarness — APPROVED** [review]
LGTM on CD

**Review by krishi0408 — APPROVED** [review]

**Review by tejakummarikuntla — DISMISSED** [review]

**Review by tejakummarikuntla — APPROVED** [review]
Sorry, I fixed a merge conflict in the CODEOWNERS file for the SCS module. This commit unintentionally reverted all previous approvals — apologies for the inconvenience.

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Review by pratmit — APPROVED** [review]
LGTM from platform side

**Review by dewan-ahmed — APPROVED** [review]
LGTM from CI

https://github.com/harness/developer-hub/pull/8585#pullrequestreview-2443607654

**Review by nofarblue — APPROVED** [review]

**Review by wei-harness — APPROVED** [review]

**Pull request merged**